### PR TITLE
Microbitv2 blogpost

### LINKED ISSUE #7646: Micro:bit V2 Blog Post
Hey Galen,
Can you get the post prepped?  We'll want to review with everyone before we publish... aiming for tomorrow.
Files are all here - https://1drv.ms/u/s!AmMIW5Hxi0RtgZ5Y2eSQvP04uJKmIg?e=d33FJ5
Thanks!
Jacqueline

### PR DESCRIPTION
**Pull request by ganicke**
Import, format, adjust images, etc. for the micro:bit V2 blogpost.

See if you like the name of the blogpost path and the size of the GIF. Both can change if you want.

Closes #7646

### CODE REVIEW

File: docs/blog/microbit/microbitv2.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+![micro:bit V2 Toolbox](/static/blog/microbit/microbitv2/v2-toolbox.png)
+
+And when you use any of the V2 blocks in your program, you’ll see the micro:bit Simulator change to represent the new micro:bit with the "V2" indicator.
+
+![micro:bit V2 Simulator](/static/blog/microbit/microbitv2/simulator.png)
+
+Web USB reliability improvements Web USB (https://makecode.microbit.org/device/usb/webusb) isn’t a new feature, but we took this opportunity to improve the reliability of this technology. Most teachers and students have told us that the download process of copying program files over to the micro:bit is one of the most painful parts of programming with the micro:bit. Web USB is a new technology (only available in Chrome and Edge browsers) which allows you to pair your micro:bit with MakeCode allowing you to do a one-click download!
```
Comment (microbit-mark) [inline docs/blog/microbit/microbitv2.md:29]:
```suggestion
## Web USB reliability improvements
Web USB (https://makecode.microbit.org/device/usb/webusb) isn’t a new feature, but we took this opportunity to improve the reliability of this technology. Most teachers and students have told us that the download process of copying program files over to the micro:bit is one of the most painful parts of programming with the micro:bit. Web USB is a new technology (only available in Chrome and Edge browsers) which allows you to pair your micro:bit with MakeCode allowing you to do a one-click download!
```

**Comment by microbit-mark** [comment]
- We should also call out the set touch mode block in Pins

image

By default, the logo is set to capacitive touch and the edge connector pins to resistive touch, but the set touch mode block enables you to change the default mode.

- Can we capitalise the V in the post filename? microbitv2.md > microbitV2.md 

Otherwise LGTM. Thanks @ganicke @Jaqster  👍

File: docs/blog/microbit/microbitv2.md (hunk 1 above)
Comment (pelikhan) [inline docs/blog/microbit/microbitv2.md:29]: Web USB -> WebUSB one word is the proper search term.

Comment (pelikhan) [reply docs/blog/microbit/microbitv2.md:29]: you can also point to https://www.youtube.com/watch?v=_-xtP2FH9B8&list=PLMMBk9hE-SepwjCAK7cY-jvq6KeQKda8x

**Review by Jaqster — APPROVED** [review]
Just added the Set Pins Mode block.  Otherwise looks good!

**Pull request merged**